apps/blueprints/src/components/ImageEditor.tsx

```diff
@@ -77,7 +77,7 @@ export const ImageEditor: React.FC<ImageEditorProps> = ({ string }) => {
         const editor = editorRef.current;
         if (!editorLoaded || !FBE || !editor) return;
 
-        const bp = await FBE.decodeToBlueprint(string);
+        const bp = await (FBE as any).decodeToBlueprint(string);
```

Comment:
I've recently did this update, but the relevant types should be in the `.yalc` directory. Please revert this change